osu.Game.Rulesets.Taiko/Beatmaps/TaikoBeatmapConverter.cs

```diff
@@ -185,7 +185,7 @@ private bool shouldConvertSliderToHits(HitObject obj, IBeatmap beatmap, IHasDist
 
             // The velocity and duration of the taiko hit object - calculated as the velocity of a drum roll.
             double taikoVelocity = sliderScoringPointDistance * beatmap.BeatmapInfo.BaseDifficulty.SliderTickRate;
-            taikoDuration = distance / taikoVelocity * beatLength;
+            taikoDuration = (int)(distance / taikoVelocity * beatLength);
```

Comment:
This change may not be necessary, let me check to see if this makes any difference.

Reply:
This one is still required:
- https://osu.ppy.sh/beatmapsets/140691#taiko/432842 (before: 1.72, after: 1.03) matches osu-stable.